### ISSUE TITLE
Missing Recipes

### ISSUE THREAD

**Issue by Jasems** [BuildCraft/BuildcraftCompat#55]
Recipes for items like basic Gate and redstone comp chipset are not available yet.  When you click on the gate it takes you to the assembly table recipe combination.  Running latest version as of 17Dec2015.

**Comment by loordgek** [comment]
for 1.8.8?

**Comment by Jasems** [comment]
Sorry I meant of the mod.  Running 1.7.10 Minecraft and 7.1.3 of BuildCraft Compat.

**Comment by AEnterprise** [comment]
the recipes in nei are not easely showable for those as they use nbt, NEI doesn't realy handle nbt differences very well (same as nei not always being able to tell the difference between the different TE machine frames)

**Comment by Jasems** [comment]
The TE machine frames you could toggle between though to see the recipe at least I think right (or it would cycle automatically I can't remember)?  Any way to make basic gate and those missing chipset recipes do that?

**Comment by AEnterprise** [comment]
you should atleast see them, look through the recipe types and they should be there

**Comment by Jasems** [comment]
If I click on "Basic Gate" for example it takes me to Integration Table and shows me stuff that makes vs the needed recipe in the Assembly Table.  There is no next page or anything to even click on to see other possible recipes.  And something like "redstone comp" chipset doesn't show any recipes at all when I click on it in NEI.

**Comment by AEnterprise** [comment]
are you using a modpack or anything that might have changed recipes?-

**Comment by Jasems** [comment]
No, it seems like it is just the assembly table recipes that are missing for our pack.  I can see all the integration table ones just fine.  Thanks for helping btw, you are extremely fast in responses!

**Comment by AEnterprise** [comment]
no problem, could you provide a complete modlist?

**Comment by ghost** [comment]
```
adv-repulsion-systems-59.0.3
buildcraft-7.1.14
buildcraft-compat-7.1.3
CodeChickenCore-1.7.10-1.0.7.47-universal
CoFHCore-[1.7.10]3.0.3-303
forestry_1.7.10-4.2.2.50
Framez2-1.7.10-0.1.1-hotfix-universal
IC2.Classic.Version.1.1.3.3
IC2NuclearControl-2.3.3a-Exist
immibis-core-59.1.2
ironchest-1.7.10-nentify-1.0-universal
MineTweaker3-1.7.10-3.0.10B
ModularPowersuits-0.11.0.293
NotEnoughItems-1.7.10-1.0.5.118-universal
Numina-0.4.0.131
OpenComputers-MC1.7.10-1.5.20.38-universal
PlayerAPI-1.7.10-1.4
ProjectRed-1.7.10-4.7.0pre9.92-Base
ProjectRed-1.7.10-4.7.0pre9.92-Compat
ProjectRed-1.7.10-4.7.0pre9.92-Integration
ProjectRed-1.7.10-4.7.0pre9.92-Lighting
ProjectRed-1.7.10-4.7.0pre9.92-World
QuiverBow_1.7.10_b101d
Railcraft_1.7.10-9.8.0.0
ReforgedBits-0.1
RenderPlayerAPI-1.7.10-1.4
RenderPlayerAPIEnhancer-1.7.10-1.4
SmartMoving-1.7.10-15.6
SmartRender-1.7.10-2.1
StorageDrawers-1.7.10-nentify-1.7.5
StorageDrawers-Forestry-1.7.10-1.1.0
Trajectory-1.7.10-0.1.2-universal
tubestuff-59.0.4
zettaindustries-1.1-106
```

**Comment by AEnterprise** [comment]
hmm i do see minetweaker in there, could you take that out for a sec and see if that helps? if it does then there's a script in the pack causing the recipe removal

**Comment by ghost** [comment]
I found the issue

```
client {
    # Should NEI facade recipes be hidden? [default: false]
    B:hideFacadeRecipes=true
}
```

Setting this to true breaks the chipset and gate (and probably other custom BC crafting) recipes.

**Comment by ghost** [comment]
It works fine if I use the config in main.cfg

```
        # Should all BC facades be shown in Creative/NEI, or just a few carefully chosen ones?
        B:facadeShowAllInCreative=false
```

So I will just leave the config in compat enabled. Is the config in compat even necessary? The config in the main file does basically the same thing.

**Comment by AEnterprise** [comment]
oh i see the issue, the compat one to hide the recipes doesn't hide the facede recipes, it prevents the registration of the assembly table recipe handler
